testfile

```diff
@@ -159,3 +161,7 @@ Inbound federation rejects remote attempts to kick local users to rooms
 An event which redacts itself should be ignored
 A pair of events which redact each other should be ignored
 Full state sync includes joined rooms
+Can list tags for a room
+Tags appear in an initial v2 /sync
+Newly updated tags appear in an incremental v2 /sync
+Deleted tags appear in an incremental v2 /sync
```

Comment:
Can you group the newly passing tests together at the bottom? And also add back the trailing new line?

Reply:
Newly passing tests _are_ at the bottom arent they?

Reply:
Ah the other 2 were up.